includes/managers/icons.php

```diff
@@ -450,9 +566,20 @@ public function __construct() {
 			add_action( 'elementor/admin/after_create_settings/' . Settings::PAGE_ID, [ $this, 'register_admin_settings' ], 100 );
 		}
 
-		add_action( 'elementor/editor/after_enqueue_styles', [ $this, 'enqueue_fontawesome_css' ] );
-		add_action( 'elementor/frontend/after_enqueue_styles', [ $this, 'enqueue_fontawesome_css' ] );
+		if ( self::is_font_icon_inline_svg() ) {
+			self::$data_manager = new Font_Icon_Svg_Data_Manager();
+
+			add_action( 'wp_footer', [ $this, 'render_svg_symbols' ], 10 );
+			add_action( 'elementor/frontend/after_enqueue_styles', function() {
+				//$css = '.elementor-widget-container .e-font-icon-svg { width: 1em; height: 1em; display: block; }';
+				$css = '';
+
+				wp_add_inline_style( 'elementor-frontend', $css );
```

Comment:
What happened here?

Reply:
Put it in the CSS files

Reply:
The commented code is redundant, so I've removed the inline style code.